package.json

```diff
@@ -2,7 +2,12 @@
   "name": "angular-blockchain-wallet",
   "version": "1.16.7",
   "description": "AngularJS front-end to My-Wallet-V3.",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "^1.6.4",
```

Comment:
Having a parts app, helper app and the main app in the same repo is starting to get a bit too confusing, but maybe just leave it here for now. cc @jtormey